modules/persistent-storage-csi-efs-cross-account.adoc

```diff
@@ -0,0 +1,304 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-efs-cross-account_{context}"]
+= AWS EFS CSI cross account support
```

Comment:
Looking at the preview, this chapter is placed after "Uninstalling the AWS EFS CSI Driver Operator" shouldn't it be placed before?

Reply:
Moved it earlier.